envision/types.py

```diff
@@ -39,6 +39,9 @@ class VehicleType(str, Enum):
     Truck = "truck"
     Trailer = "trailer"
     Car = "car"
+    Pedestrian = "pedestrian"
+    Motorcycle = "motorcycle"
+    Passenger = "passenger"
```

Comment:
What will Envision do with these if they're used?   Will it display them properly?

Reply:
No this won't work, this was a mistake on my end about how to go about solving the problem